Update README.md (#1)
JuulLabs-OSS/kotlin-guides: README.md

### View IDs

> View ID names (`android:id` in XML) should be all lowercase with underscores separating words (snake_case).  It is required that every name will have at least one underscore that way it is distinguishable from local variables. In instances were it proves difficult, then as a last resort, it should be fine to postfix the name with the component type (e.g. _button or _view).

Reviewer: ```suggestion
View ID names (`android:id` in XML) should be all lowercase with underscores separating words (snake_case). It is required that every name have at least one underscore to distinguish it from local variables. In instances were it proves difficult, then name may be postfixed with the component type (e.g. `_button` or `_view`).
```

### View IDs

View ID names (`android:id` in XML) should be all lowercase with underscores separating words (snake_case).  Do your best to create a name that contains an underscore so it is distinguishable from local variables. 

```xml
<android.support.v7.widget.RecyclerView
    android:id="@+id/dance_moves"
    />

<Button
    android:id="@+id/login"
    />
```

> When using [Kotlin Android Extensions] to provide synthetic view properties, use the `as` keyword on [imports] to name

Reviewer 2: take out `as`

> properties that do not have an underscore in their name, and append `_view` to the end

Reviewer 3: @TravisJuul I thought you kinda agree with me on this one... or at least cringed  😛 

Either you should go with all views having "_<type>" or none. This feels like a hack :)

```kotlin
import kotlinx.android.synthetic.main.scan.dance_moves

> import kotlinx.android.synthetic.main.scan.login as login_view

Reviewer 4: I think one of the benefits of going with the XML name is that we no longer have to go through the trouble of specifying `as` for the `import`.

So perhaps the requirement for having an `_` should be for the XML `id`?

    android:id="@+id/dance_moves"
    />
```
The name should be [snake case].  Do your best to create a name that contains an underscore so it is distinguishable from local variables.  If a name cannot be created with an underscore, then postfix [_view] at the end.

```kotlin
import kotlinx.android.synthetic.main.scan.dance_moves

> import kotlinx.android.synthetic.main.scan.login as login_view

Reviewer 2: I was thinking we'd always use at least one `_` in the xml
